### PR TITLE
Finished basic models

### PR DESCRIPTION
**Pull request by Asphahyre**
Finished two prediction models (classification tree and regression tree) and added documentation for both.

Added review of the work done until now in `PROJECT.md`.

### CODE REVIEW

File: modeles/PROJECT.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Premises of my prediction model
+
+As I learned how to manipulate data, and how sensors are proceeding their measures, I can know aggregate the first features of my model.
+
+Here the list of the features I tried and the way I imagined it will affect the prediction:
+
+Feature | Expected effect
+--------|----------------
+Season (current month) | I think there is societal factors varying the need to go to a recycle center depending on the period of the year. An example can be the sprint cleaning. I also think that recycle center's usage isn't the same during summer holidays than the rest of the year.
```
Comment (vallettea) [inline modeles/PROJECT.md]: there are societal factors and we say recycling center

File: modeles/PROJECT.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Day of the week | I think people are going to a recycle center depending on their week's schedule.
+Hour of the day | For the same reason, and maybe also because people don't really need to throw their waste at midnight.
+Opening hours of the center | Because people need to respect the law more than throwing their waste, I think.
+Weather | Because I think people won't go to a recycle center during a snowstorm, or even a rainstorm, and because they would prefer to wait to a cleared weather than go during the worst weather.
+Previous hour's affluence | Because I think they're common factors I didn't found that are making people needing to throw waste at same periods of the day. Admitting we had an important affluence since the beginning of the day, we can think we would really probably have an important affluence until the rest of this day.
+
+For the features that didn't really improve the model, we have the *weather* feature. The reason I found is the reason we argued against me: people are going to a recycle center when they need it. Thus, predicting the affluence of a recycle center correspond to predict when people will need to throw their waste.
```
Comment (vallettea) [inline modeles/PROJECT.md]: The reason I found is the reason we argued against me ???!? very unclear and heavy

you speak too much about yourself, don't say "I think" all the time, try to be as objective as you can, people know you are the author.

**Comment by vallettea** [comment]
good work (except for the english).
Can you had more quantitative results (confusion matrix at each steps of teh classification so we can see what makes you think some features work and other don't + pearson and rmse for the regression).

**Comment by Asphahyre** [comment]
Ok, I didn't really know what you expected in the form of this review.
I'll correct it.

**Comment by vallettea** [comment]
No problem, that's why PR are for.
We'll correct the english at the end when will write an article.

**Pull request merged**